### PR TITLE
feat(ftux): show back button on status page when there's a failure

### PR DESCRIPTION
**Pull request by neurosnap**
We also allow the user to make changes to environment variables and service commands without re-provisioning databases.

We also require user to provide a database handle instead of generating one for them.

https://user-images.githubusercontent.com/1940365/221024042-44da351e-be69-45f0-8056-9aa0a87e9df2.mp4

### CODE REVIEW

File: src/ui/pages/create-project-git.tsx [hunk 1]
```diff
@@ -366,26 +373,33 @@ const validateDbs = (
...
 
   const validate = (item: TextVal) => {
-    const imgs = dbImages.filter((img) => img.type === item.key);
+    if (!/^[0-9a-z._-]{1,64}$/.test(item.key)) {
+      errors.push({
+        item,
+        message: `[${item.key}] is not a valid handle: /\A[0-9a-z._-]{1,64}\z/`,
+      });
+    }
+    const [type = "", version = ""] = item.value.split(":");
```
Comment (madhuravius) [inline src/ui/pages/create-project-git.tsx:382]: nit but might be useful to not use a reserved word `type`

File: src/ui/pages/create-project-git.tsx [hunk 2]
```diff
@@ -555,41 +603,50 @@ export const CreateProjectGitSettingsPage = () => {
...
+                {existingEnvStr ? (
+                  <p>
+                    Databases have already been created so you cannot make
+                    changes to them in this screen anymore.
+                  </p>
+                ) : (
+                  <>
+                    <p>
+                      You can provide as many databases as you need (each line
+                      is a separate database, e.g. NAME=DB:VERSION).
```
Comment (madhuravius) [inline src/ui/pages/create-project-git.tsx:621]: might wanna put that `NAME=DB:VERSION` in a pre tag

File: src/ui/pages/create-project-git.tsx [hunk 3]
```diff
@@ -619,15 +673,19 @@ export const CreateProjectGitSettingsPage = () => {
             label="Service and Commands"
             htmlFor="commands"
             feedbackVariant="info"
+            description={
+              <div>
+                <p>
+                  Each line is separated by a service command in format:
```
Comment (madhuravius) [inline src/ui/pages/create-project-git.tsx:679]: same as above for the code/formatted text

File: src/ui/pages/create-project-git.tsx (hunk 2 above)
Comment (neurosnap) [reply src/ui/pages/create-project-git.tsx:621]: @joeclayallday said to remove those

File: src/ui/pages/create-project-git.tsx (hunk 3 above)
Comment (neurosnap) [reply src/ui/pages/create-project-git.tsx:679]: @joeclayallday said to remove those

**Review by madhuravius — APPROVED** [review]

**Pull request merged**